external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisBackedBlockRDD.scala

```diff
@@ -36,7 +36,8 @@ import org.apache.spark.util.NextIterator
 /** Class representing a range of Kinesis sequence numbers. Both sequence numbers are inclusive. */
 private[kinesis]
 case class SequenceNumberRange(
-    streamName: String, shardId: String, fromSeqNumber: String, toSeqNumber: String)
+    streamName: String, shardId: String, fromSeqNumber: String, toSeqNumber: String,
+    recordCount: Int)
```

Comment:
Why is this a property of a range -- or when would it not equal (from - to + 1)?